mmdet/utils/deployment/symbolic.py

```diff
@@ -20,10 +20,45 @@
 from torch.onnx.symbolic_registry import register_op, get_registered_op, is_registered_op
 
 
-def py_symbolic(op_name=None, namespace='mmdet_custom'):
+def are_kwargs_contain_tensors(**kwargs):
+        for elem in kwargs.values():
+            if isinstance(elem, (list, tuple, torch.Tensor)):
+                return True
+        return False
+
+
+def py_symbolic(op_name=None, namespace='mmdet_custom', outside_adapter=None):
+    """
+    The py_symbolic decorator allows you to wrap your custom function in 
+    torch.autograd.Function and associate it with a custom symbolic function
+    to represent the custom operation in a computation graph.
```

Comment:
```suggestion text
    The py_symbolic decorator allows associating a function with a custom symbolic function
    that defines its representation in a computational graph.
```

IMO, usage of `torch.autograd.Function` is more related to implementation details.